wren-ai-service/src/pipelines/retrieval/sql_functions.py

```
@@ -0,0 +1,173 @@
+import logging
+import sys
+from typing import Any, Dict, List, Optional
+
+import aiohttp
+from cachetools import TTLCache
+from hamilton import base
+from hamilton.async_driver import AsyncDriver
+from hamilton.function_modifiers import extract_fields
+from langfuse.decorators import observe
+from haystack.document_stores.types import DocumentStore
```

Ruff (0.8.2)

`haystack.document_stores.types.DocumentStore` imported but unused

Remove unused import: `haystack.document_stores.types.DocumentStore`

(F401)